recipe/meta.yaml

```diff
@@ -1,41 +1,37 @@
     patches:
       - patches/0001-Add-Anaconda-Distribution-version-logic.patch
       - patches/0002-Darwin-Add-arch-x86_64-to-UnicCCompiler-ld_args.patch
-      - patches/0003-Win32-distutils-Also-look-for-executable.bat.patch
-      - patches/0004-Win32-Change-FD_SETSIZE-from-512-to-2048.patch
-      - patches/0005-Win32-distutils-Add-support-to-cygwinccompiler-for-V.patch
-      - patches/0006-Do-not-pass-g-to-GCC-when-not-Py_DEBUG.patch
-      - patches/0007-Support-cross-compiling-byte-code.patch
-      - patches/0008-Win32-Fixes-for-Windows-GCC-interop-needed-by-RPy2-a.patch
-      - patches/0009-Darwin-Look-in-sysroot-usr-lib-include-if-sysroot-is.patch
-      - patches/0010-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch
-      - patches/0011-Win32-Do-not-download-externals.patch
-      - patches/0012-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch
-      - patches/0013-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
-      - patches/0014-Disable-new-dtags-in-unixccompiler.py.patch
-      - patches/0015-Fix-cross-compilation-on-Debian-based-distros.patch
-      - patches/0016-Disable-registry-lookup-unless-CONDA_PY_ALLOW_REG_PA.patch
-      - patches/0017-Unvendor-openssl.patch
-      - patches/0018-Unvendor-sqlite3.patch
-      # https://bugs.python.org/issue35644
-      - patches/0019-venv-Revert-a-change-from-https-github.com-python-cp.patch
-      # undefined reference to `CondaEcosystemGetWarnings'
-      # - patches/0020-Add-CondaEcosystemModifyDllSearchPath.patch
+      - patches/0003-Win32-Change-FD_SETSIZE-from-512-to-2048.patch
+      - patches/0004-Win32-distutils-Add-support-to-cygwinccompiler-for-V.patch
+      - patches/0005-Do-not-pass-g-to-GCC-when-not-Py_DEBUG.patch
+      - patches/0006-Support-cross-compiling-byte-code.patch
+      - patches/0007-Win32-Fixes-for-Windows-GCC-interop-needed-by-RPy2-a.patch
+      - patches/0008-Darwin-Look-in-sysroot-usr-lib-include-if-sysroot-is.patch
+      - patches/0009-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch
+      - patches/0010-Win32-Do-not-download-externals.patch
+      - patches/0011-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch
+      - patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
+      - patches/0013-Disable-new-dtags-in-unixccompiler.py.patch
+      - patches/0014-Disable-registry-lookup-unless-CONDA_PY_ALLOW_REG_PA.patch
+      - patches/0015-Unvendor-openssl.patch
+      - patches/0016-Unvendor-sqlite3.patch
+      - patches/0017-venv-Revert-a-change-from-https-github.com-python-cp.patch
+      # TODO re-enable
+      #- patches/0018-Add-CondaEcosystemModifyDllSearchPath.patch
   # TODO :: Depend on our own packages for these:
   - url: https://github.com/python/cpython-source-deps/archive/xz-5.2.2.zip          # [win]
     folder: externals/xz-5.2.2                                                       # [win]
```

Comment:
In the build-portion of the recipe, there's still a reference to `python3.7` under `no_link`.  I (naively) tried to parametrize this, c.f. https://github.com/conda-forge/python-feedstock/pull/261/commits/d1cf4a231ceaeef5a4e30d2ff719bec5fa68fb34.